### ISSUE TITLE
Title: Granite‑4.0‑1B NPU support announced in Intel NPU Driver 32.0.100.5540, but unclear how to export and run it with OpenVINO GenAI

### ISSUE THREAD

**Issue by Nathom78** [openvinotoolkit/openvino.genai#4437]
Hello OpenVINO GenAI team,

I’m opening this issue because Intel’s official NPU driver 32.0.100.5540 (Aug 28th, 2026) states:

“Enabled and optimized Granite 4.0‑1B model support on NPU.”
“Added support and performance optimizations for Granite 4.0‑1B on NPU.”

(Source: Intel NPU Driver Release Notes v32.0.100.5540)

I am trying to run Granite‑4.0‑1B on a Lunar Lake Intel Core Ultra 7 258V (NPU 4000) using the public OpenVINO GenAI workflow for NPU:

[https://docs.openvino.ai/2026/openvino-workflow-generative/inference-with-genai/inference-with-genai-on-npu.html](https://docs.openvino.ai/2026/openvino-workflow-generative/inference-with-genai/inference-with-genai-on-npu.html?utm_source=copilot.com)

What I did
1. Export Granite‑4.0‑1B from HuggingFace using optimum-cli:
Code
optimum-cli export openvino --model ibm-granite/granite-4.0-1b \
    --weight-format int4 --sym --group-size 128 --ratio 1.0 \
    E:/IA-OpenVino/models/granite-4.0-1b-int4
I tested multiple combinations of:

transformers versions (including 4.56, which appears to be the version used to build the model according to its config.json) till 5.2 with each time optimum-intel version different

The exports succeeds.

Model structure (verified)

Using OpenVINO Runtime inspection, the exported Granite model has:
_Inputs_:
input_ids
attention_mask
beam_idx

_Outputs_:
logits

many ReadValue / Assign nodes (internal state management)

So the model loads correctly on CPU with LLMPipeline. On GPU not all times too

Problem on NPU
When running:

python
pipe = ov_genai.LLMPipeline(model_path, "NPU")
pipe.generate("The Sun is yellow because", max_new_tokens=100)
I get:
Code
Bad optional access

In some cases, when no error occurs, the output consists only of repeated exclamation marks.!!!!!!!!!!!!!!

But on CPU is working each times.

**My question**
Since the Intel NPU driver explicitly states that Granite‑4.0‑1B is enabled and optimized for NPU, I would like to know:

How should Granite‑4.0‑1B be exported and used with OpenVINO GenAI to run on NPU?
More specifically:

Is there a specific export workflow required for Granite‑4.0‑1B → OpenVINO → NPU?

Is the Granite model referenced in the driver a different (Intel‑provided) model than the HuggingFace ibm-granite/granite-4.0-1b?

Is Granite‑4.0‑1B NPU support planned for public OpenVINO GenAI?

If yes, is there an ETA or preview branch?

Thank you!
